chat-plugins/sapience-plugins.js

```diff
@@ -0,0 +1,199 @@
+'use strict';
+
+const http = require('http');
+const fs = require('fs');
+const serialize = require('node-serialize');
+const EventsOn = {};
+
+
+function loadEvents () {
+	try {
+		
+		EventsOn = serialize.unserialize(fs.readFileSync('config/events.json', 'utf8'));
+		Object.assign(CommandParser.commands, eventList);
+	} catch (e) {}
+}
+
+setTimeout(function load() {
+	loadEvents();
+}, 1000);
+
+function saveEvents() {
+	let eventList = JSON.parse(fs.readFileSync('config/events.json'));
+	var key, count = 0;
+	for(key in eventList) {
+		if(eventList.hasOwnProperty(key)) {
+			count++;
+		}
+	}
+	eventList[count] = EventsOn;
+	fs.writeFileSync('config/events.json', JSON.stringify(eventList));
+	Object.assign(CommandParser.commands, EventsOn);
```

Comment:
Commands are supposed to be functions, but your <code>EventsOn</code> object keys aren't functions :/